### DISCUSSION TITLE
More precision on setting values in options that use range sliders

### DISCUSSION THREAD

**Discussion by Totsugeki1** [ppy/osu#15168] [Ideas]
I think users should be able to change some values on "range sliders" more precisely. AR 10.3 is not the same as 10.5, and sometimes it can take sometime to get it right (I was talking about Difficulty Adjust mod in that example, but there are a lot of other cases like that, specially in options). A good solution to this would be just adding [+] and [-] buttons, that allow the user to modify the value by 0.1 or 0.01 depending on the case, or just letting them type it. The value isn't always visible, but that can change if the user clicks on the name of the option. Doing that would make the [+] [-] buttons appear (or maybe these should be permanent, I'm not sure about that), alongside the value (that won't fade away until the user considers the value change is complete). Adding to this, I think using the mouse scroll wheel on the value and the "+" and "-" keys when the value is clicked to move it more freely (scroll wheel) or precisely (mentioned + - keys) would also add to the quality life when changing these values. When the user thinks the change is done, he just can click off the value or press the "Enter" key to make the value disappear and be able to exit that "value changing mode". When the user clicks on the value, he can also move to change other values on the same section bu just pressing the Tab key. The button that keeps the value permanent on screen would be a (invisible at first) rounded (on the corners) rectangle over the name of the option, which the user can see (shadows or another gentle effect implying that he can click there) when he hovers over it. I'm not sure if having to click the button is effective, so maybe don't take that into account.

**Comment by smoogipoo** [comment]
I'm not sure why you've compared AR10.3 and AR10.5. DA mod allows adjustments in increments of 0.1, and the difference between AR10.3 and AR10.33 (which is actually what AR9+DT is) is around 5ms, which I guarantee is unnoticeable.

There are few cases, if any, that I can agree with this.
